megazords/ios-rust/src/lib.rs

```diff
@@ -21,3 +23,14 @@ pub use sync15;
 pub use sync_manager;
 pub use tabs;
 pub use viaduct_reqwest;
+
+use nss::ensure_initialized;
+
+// TODO: would be nice if we could write
+//  #[uniffi::export]
+//  pub use nss::ensure_initialized as ensure_nss_initialized;
+// instead.
+#[uniffi::export]
+pub fn ensure_nss_initialized() {
+    ensure_initialized()
+}
```

Comment:
a) I wonder if this should just be `initialize` and it could handle any initialization we needed (like `Megazord.init()` on Android).

b) Whatever change you make, it could be good to add a prep PR that adds the function, but doesn't do anything.  Then we could get the consumer applications to call this function before merging the actual breaking changes.  That removes the stress of having to resolve the breaking change the day the nightly is released.

Reply:
I really like the idea :100: will do

Reply:
https://github.com/mozilla/application-services/pull/6607 has initialize function stubs